storage-proofs/porep/src/stacked/vanilla/memory_handling.rs

```diff
@@ -76,8 +76,8 @@ impl<T: FromByteSlice> CacheReader<T> {
     // This is unused, but included to document the meaning of its components.
     // This allows splitting the reset in order to avoid a pause.
     pub fn reset(&self) -> Result<()> {
-        self.start_reset();
-        self.finish_reset();
+        self.start_reset()?;
+        self.finish_reset()?;
```

Comment:
I have a couple PRs that fix this more concisely (e.g. https://github.com/filecoin-project/rust-fil-proofs/pull/1299/files#diff-d3aadd2342500c276d83eddd08e5a415)